.github/dependabot.yml

```diff
@@ -0,0 +1,195 @@
+version: 2
+registries:
+  maven-repository-nexus-intern-sda-se-online-repository-sda-se-re:
+    type: maven-repository
+    url: https://nexus.intern.sda-se.online/repository/sda-se-releases
+    username: SDABot
+    password: "${{secrets.MAVEN_REPOSITORY_NEXUS_INTERN_SDA_SE_ONLINE_REPOSITORY_SDA_SE_RE_PASSWORD}}"
+  maven-repository-nexus-intern-sda-se-online-443-repository-sda-s:
+    type: maven-repository
+    url: https://nexus.intern.sda-se.online:443/repository/sda-se-public/
+    username: depend-pub-read
+    password: "${{secrets.MAVEN_REPOSITORY_NEXUS_INTERN_SDA_SE_ONLINE_443_REPOSITORY_SDA_S_PASSWORD}}"
+  maven-repository-nexus-sda-se-io-443-repository-sda-se-public:
+    type: maven-repository
+    url: https://nexus.sda-se.io:443/repository/sda-se-public/
+    username: depend-pub-read
+    password: "${{secrets.MAVEN_REPOSITORY_NEXUS_SDA_SE_IO_443_REPOSITORY_SDA_SE_PUBLIC_PASSWORD}}"
+  maven-repository-nexus-intern-sda-se-online-repository-sda-se-pu:
+    type: maven-repository
+    url: https://nexus.intern.sda-se.online/repository/sda-se-public/
+    username: depend-pub-read
+    password: "${{secrets.MAVEN_REPOSITORY_NEXUS_INTERN_SDA_SE_ONLINE_REPOSITORY_SDA_SE_PU_PASSWORD}}"
+  maven-repository-nexus-sda-se-io-repository-sda-se-public:
+    type: maven-repository
+    url: https://nexus.sda-se.io/repository/sda-se-public/
+    username: depend-pub-read
+    password: "${{secrets.MAVEN_REPOSITORY_NEXUS_SDA_SE_IO_REPOSITORY_SDA_SE_PUBLIC_PASSWORD}}"
+
+updates:
+- package-ecosystem: gradle
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "02:00"
+    timezone: Europe/Berlin
+  open-pull-requests-limit: 33
+  target-branch: master
+  ignore:
```

Comment:
We could limit `org.apache.kafka:kafka-clients` and `org.apache.kafka:kafka_2.12 ` to `"< 3"` or `"< 6"` to get useful updates and not get the version numbers from confluent.

Reply:
Since it shouldn't ask the confluent registry anymore, this shouldn't be needed. Maven Central only contains the correct versions.